docs/contents/releases.md

```diff
@@ -0,0 +1,90 @@
+# Releases
+
+We use the following different release statuses:
+
+* [PROPOSED](#proposed)
+* [PLANNED](#planned)
+* [BUILD](#build)
+* [DEVELOPER PREVIEW](#developer-preview)
+* [BETA](#beta)
+* [GENERALLY AVAILABLE](#generally-available) (GA)
+
+## PROPOSED
+
+The `PROPOSED` status indicates that **the AWS service is on our radar** for
+building into ACK.
+
+There will be a Github Issue for tracking the build of the ACK service
+controller for the service.
+
+The GitHub Issue **WILL NOT** be associated with a GitHub Milestone.
+
+## PLANNED
+
+The `PLANNED` status indicates that the AWS service is on our radar for
+building into ACK.
+
+There will be a Github Issue for tracking the build of the ACK service
+controller for the service.
+
+The GitHub Issue **WILL BE** associated with a GitHub Milestone indicating the
+target date for the `DEVELOPER PREVIEW` release of that service's controller.
+
+## BUILD
```

Comment:
we should call out we will define which resources are in for beta/dev and for GA